exporter/stats/stackdriver/stackdriver.go

```diff
@@ -87,6 +87,13 @@ type Options struct {
 	// can be buffered before batch uploading them to the backend.
 	// Optional.
 	BundleCountThreshold int
+
+	// Resource is an optional field that represents the Stackdriver
+	// MonitoredResource, a resource that can be used for monitoring.
+	// If no custom ResourceDescriptor is set, a default MonitoredResource
+	// with type global and no resource labels will be used.
+	// Optional.
```

Comment:
Oops, we forgot to remove this, apologies, I should have spelled that out in my previous request :)

Reply:
^since we now explicitly spell out that Resource is an optional field, or I guess that's the style of the repo?

Reply:
Oh I saw other fields also have this "Optional" word, do you mean we need to remove these?

Reply:
Nah, it's alright then, LGTM.